File: schedule_sample/sample/sample.swift
func if_test(num : Int){
    if(num > 100){
      print("100 < num\n");
    }else if(num > 50){
      print("50 < num <= 100\n");
    }else if(num > 0){
      print("0 < num <= 50\n");
    }else if(num == 0){
      print("num < 0>\n");
    }else{
      print("num < 0\n");
    }
}

if_test(num :1000);
// 100 < num

if_test(num :70)
// 50 < num <= 100

if_test(num :0)
// num == 0

if_test(num :-100)
// num < 0

File: schedule_sample/sample/sample.js
/*
  javascript

  webブラウザの動きを作ることができる

*/

function if_test($num){
  if($num > 100){
    console.log("100 < num");
  }else if($num > 50){
    console.log("50 < num <= 100");
  }else if($num > 0){
    console.log("0 < num <= 50");
  }else if($num === 0){
    console.log("num < 0");
  }else{
    console.log("num < 0");
  }
}

if_test(1000);
 // 100 < num

if_test(70);
 // 50 < num <= 100

if_test(0);
 // num == 0

if_test(-100);
 // num < 0
